core/src/main/java/org/elasticsearch/action/admin/indices/create/CreateIndexResponse.java

```diff
@@ -30,22 +31,43 @@
  */
 public class CreateIndexResponse extends AcknowledgedResponse {
 
+    private boolean timedOutWaitingForShards;
+
     protected CreateIndexResponse() {
     }
 
-    protected CreateIndexResponse(boolean acknowledged) {
+    protected CreateIndexResponse(boolean acknowledged, boolean timedOutWaitingForShards) {
         super(acknowledged);
+        this.timedOutWaitingForShards = timedOutWaitingForShards;
     }
 
     @Override
     public void readFrom(StreamInput in) throws IOException {
         super.readFrom(in);
         readAcknowledged(in);
+        timedOutWaitingForShards = in.readBoolean();
     }
 
     @Override
     public void writeTo(StreamOutput out) throws IOException {
         super.writeTo(out);
         writeAcknowledged(out);
+        out.writeBoolean(timedOutWaitingForShards);
+    }
+
+    /**
+     * Returns true if the request timed out waiting for the required number
+     * of active shards to be started.  If the index was not successfully
```

Comment:
active shards cannot be started. They are started by definition of being active ;-)